Translators/Incident/README.md

```diff
@@ -0,0 +1,53 @@
+# Incident Translator
+
+## Table of Contents
+
+- [About](#about)
+- [Getting Started](#getting_started)
+- [Usage](#usage)
+- [Contributing](../CONTRIBUTING.md)
+
+## About <a name = "about"></a>
+
+Python Incident to TIM message translator that is designed to pull messages from the CDOT Incidents feed and translate to TIM messages.

```

Comment:
suggestion: it would be great to include sample input and output files so we know what to expect from the application

Reply:
I've added sample input/output files